### PR TITLE
Write new db seed script

### PR DESCRIPTION
**Pull request by topocount**
This new seed script checks every box in https://github.com/coordinape/coordinape/issues/692 and is meant to be at
least a little bit more maintainable and extensible than its
predecessor.

Nominees were omitted but can be added if desired. I figured the
workflow for testing vouching and nominees is simple enough that it's
not worth setting something up mid-stream.

Avatars are now generated as a part of the seed after running `yarn docker:clean`

All accounts used in this seed can be accessed by utilizing the seed
`test test test test test test test test test test test junk` in one's
wallet.

test plan:

- yarn docker:clean
- yarn docker:start
- yarn db-seed-fresh

Observe that all details in https://github.com/coordinape/coordinape/issues/692 are taken care of

### CODE REVIEW

**Comment by META-DREAMER** [comment]
FWIW I like using https://www.getsynth.com/ for stuff like this, check it out and see if it tickles your fancy

Also there's this, not sure if its helpful here but if we have zod schemas for stuff would be nice to reuse for generating mock data as well: https://github.com/anatine/zod-plugins/tree/main/libs/zod-mock

File: scripts/db_seed_test.ts [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+async function createCircleWithPendingGifts() {
+  const result = await insertMemberships(
+    getMembershipInput({ protocolInput: { name: 'Open Epoch With Gifts' } }, {})
+  );
+  const circleId = result[0].circle_id;
+  const epochId = await makeEpoch(
+    circleId,
+    DateTime.now().minus({ days: 6 }),
+    DateTime.now().plus({ hours: 6 }),
```
Comment (levity) [inline scripts/db_seed_test.ts:132]:
```suggestion
    DateTime.now().plus({ days: 14 }),
```

**Review by levity — CHANGES_REQUESTED** [review]
I don't want to context-switch rn to test the output of this, so here's some feedback just on the code. Happy to just merge it once the issues below are addressed and then tweak it if necessary later on.

File: scripts/db_seed_test.ts (hunk 1 above)
Comment (topocount) [reply scripts/db_seed_test.ts:132]: @levity this epoch is supposed to be close to ending.

File: scripts/util/seed.ts [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+  input: MemberInput,
+  epochId: number,
+  pending = true,
+  userIdx = 0,
+  memberSlice = [1, 6]
+) {
+  const amount = 5;
+  const user = input[userIdx];
+  const members = input.slice(...memberSlice);
+  for (let i = 0; i < members.length; i++) {
```
[the remark this answers is not in the document]
Comment (topocount) [reply scripts/util/seed.ts:253]: @levity I wanted to use `for...of` syntax here but I need to use the index to generate timestamps.

File: scripts/db_seed_test.ts (hunk 1 above)
Comment (levity) [reply scripts/db_seed_test.ts:132]: ah, i see. i was thinking of a different use case, for generating a current epoch that sticks around for a while for use in development. never mind!

**Review by levity — APPROVED** [review]

**Pull request merged**